### PR TITLE
CO-95 - updated-roles

### PR DESCRIPTION
**Pull request by luis-munoz**
- updated minimal roles for fusion installation
- added documentation

### CODE REVIEW

File: install-roles/README.md [hunk 1]
```diff
@@ -3,23 +3,60 @@
...
+        crd:
+          create: false
+
+    argo:
+      createAggregateRoles: false
+
+    ml-model-service:
+      runLabelNamespaceJob: false
+    ```
+
```
Comment (ian-thebridge-lucidworks) [inline install-roles/README.md:59]: Can we add these as another example values file and update the docs to add the files as a values file? Makes it easier for the end user than having to modify multiple keys in a values file.

File: install-roles/README.md [hunk 2]
```diff
@@ -3,23 +3,60 @@
...
+* Modify your upgrade fusion script and add `--skip-crds` to the helm command.
```
Comment (ian-thebridge-lucidworks) [inline install-roles/README.md:60]: Can we add support to the upgrade script for this parameter, so a user doesn't have to modify it?

File: install-roles/cluster-role.yaml [hunk 3]
```diff
@@ -5,58 +5,6 @@ kind: ClusterRole
 metadata:
```
Comment (ian-thebridge-lucidworks) [inline install-roles/cluster-role.yaml:5]: Can we add a comment at the top of this file that says which versions it is applicable to? 
```
These minimal values files are valid with fusion version:
  5.3.0
 .... etc
```

File: install-roles/README.md [hunk 4]
```diff
@@ -3,23 +3,60 @@
 This directory contains the minimal role and clusterRole that must be assigned to
 a user in order to install fusion 5 with helm v3.
 
+First, use the `customize_fusion_values.sh` script for creating your fusion values yaml file.
+
 To use these role in a cluster, as an admin user first create the namespace that you wish to
-install fusion into:
+install fusion and label it into:
```
Comment (ian-thebridge-lucidworks) [inline install-roles/README.md:9]:
```suggestion
install fusion into and label it for seldon's use:
```

File: install-roles/README.md [hunk 5]
```diff
@@ -3,23 +3,60 @@
...
+helm fetch lucidworks/fusion --version <fusion-version> --untar
+find . -iname crds -type d -exec kubectl apply -f {} \;
 ```
+
 Apply the `role.yaml` and `cluster-role.yaml` files to that namespace
 
 ```
-k apply -f cluster-role.yaml
-k apply -f --namespace fusion-namespace role.yaml
+kubectl apply -f cluster-role.yaml
```
Comment (ian-thebridge-lucidworks) [inline install-roles/README.md:26]: Do we use `k` in the rest of the documentation in this repo? Should we keep it as k for consistency?

**Review by ian-thebridge-lucidworks — CHANGES_REQUESTED** [review]

File: install-roles/cluster-role.yaml (hunk 3 above)
Comment (luis-munoz) [reply install-roles/cluster-role.yaml:5]: Added comments to role and clusterroles files

File: install-roles/README.md (hunk 4 above)
Comment (luis-munoz) [reply install-roles/README.md:9]: Updated readme

File: install-roles/README.md (hunk 5 above)
Comment (luis-munoz) [reply install-roles/README.md:26]: Updated readme replacing k with kubectl

File: install-roles/README.md (hunk 1 above)
Comment (luis-munoz) [reply install-roles/README.md]: Moved these values to a separate yaml file

Comment (luis-munoz) [reply install-roles/README.md]: Also updated the Readme file for using it as extra vaues file

File: install-roles/README.md (hunk 2 above)
Comment (luis-munoz) [reply install-roles/README.md]: I have added a new option for the upgrade script

**Review by ian-thebridge-lucidworks — APPROVED** [review]
LGTM

**Pull request merged**